source/jsonnet/england-wales/ccs/blocks/individual/employment_status.jsonnet

```diff
@@ -4,16 +4,10 @@ local rules = import 'rules.libsonnet';
 local question(title) = {
   id: 'employment-status-question',
   title: title,
+  description: 'Include casual or temporary work, even if only for one hour',
```

Comment:
Should all of these descriptions end in full stops or not? I can see they are different in the spreadsheet, should they be the same. cc @donnap573

Reply:
It's a good question. We do seem to be erratic with full stops, and the spreadsheet doesn't have them